libraries/Microsoft.Bot.Configuration/Services/LuisService.cs

```diff
@@ -52,6 +52,16 @@ public LuisService()
         /// <returns>endpoint url</returns>
         public string GetEndpoint()
         {
+            var region = this.Region.ToLower();
```

Comment:
I would add comments stating why we do this so future maintainers can understand it.